src/object_store/src/object/opendal_engine/azblob.rs

```diff
@@ -35,7 +38,14 @@ impl OpendalObjectStore {
         builder.endpoint(&endpoint);
         builder.account_name(&account_name);
         builder.account_key(&account_key);
-        let op: Operator = Operator::new(builder)?.finish();
+        let op: Operator = Operator::new(builder)?
+            .layer(
+                RetryLayer::new()
+                    .with_factor(2.0)
+                    .with_min_delay(Duration::from_secs(1))
+                    .with_max_times(4),
```

Comment:
Does this include the initial attempt? In other words, does this mean retry for 3 times or 4 times? For s3 object store, we use `with_max_attempts(4)` and it includes the initial attempt.

Reply:
Oh, nice catch, we should set `max_times = 3.`